test/cookbooks/lxctests/recipes/simple.rb

```diff
@@ -1,5 +1,11 @@
 require 'chef/provisioning'
+require 'chef/provisioning/lxc_driver'
+::Chef::Config.from_file '/home/ranjib/workspace/foss/chef-provisioning-lxc/.chef/knife.rb'
 with_driver 'lxc'
 # default ubuntu template will install 14.04, where chef is not well tested, lets use 12.04
-with_machine_options :template => 'ubuntu', :template_options=>['-r','precise']
-machine 'simple'
+machine 'simple' do
+  machine_options(
+    template: 'download',
```

Comment:
we should make this default. download template uses pre-baked tarballs instead of using debootstrap or yum to construct the container on the fly. Its fast, and does not require root permission (creating the /dev devices). where can i declare defaults for these values? driver?